### PR TITLE
add script to enable mathjax and enable inline math

### PR DESCRIPTION
**Pull request by chblw**
This script is needed to enable math mode in markdown documents. I put it in footer since it's what is recommended, as all posts have this footer.

### CODE REVIEW

**Comment by stecaron** [comment]
For the @samperochkin post (stein-baseball) I used

**Comment by chblw** [comment]
Yes, I followed mathjax's procedure. stein still displays properly with merge. Inline math is only activated on posts and not on book reviews.

**Comment by davebulaval** [comment]
From other [source](http://haixing-hu.github.io/programming/2013/09/20/how-to-use-mathjax-in-jekyll-generated-github-pages/) I didn't got the same code.

File: themes/hugo-future-imperfect/layouts/partials/footer.html [hunk 1]
```diff
@@ -38,8 +38,23 @@
         {{ end }}
       {{ end }}
     {{ end }}
-
+    
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```
Comment (davebulaval) [inline themes/hugo-future-imperfect/layouts/partials/footer.html]: `<script type="text/javascript"
        src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>`

Comment (davebulaval) [reply themes/hugo-future-imperfect/layouts/partials/footer.html]: Should not we add the `mathjax-style.css` also ?

Comment (chblw) [reply themes/hugo-future-imperfect/layouts/partials/footer.html]: I only added what I need to fix the missing math.

Comment (davebulaval) [reply themes/hugo-future-imperfect/layouts/partials/footer.html]: allright.

**Comment by samperochkin** [comment]
C'est all good ça si je comprends bien?

**Comment by chblw** [comment]
yes j'ai merge master et c'est all good

**Comment by davebulaval** [comment]
Et fermer la PR ?

**Pull request closed without merging**